src/components/MFCard/DynamicCardIframe.tsx

```diff
@@ -40,8 +65,8 @@ const DynamicCardIframe: React.FC<Props> = ({ task, hash }) => {
             iframe?.contentWindow?.location.reload();
           }
         })
-        .catch((err) => {
-          console.error('Error fetching dynamic card data: ', err);
+        .catch((_) => {
+          // console.error('Error fetching dynamic card data: ', err);
```

Comment:
Is this deliberate?

Reply:
yes. because the server can now throw 404s for a few initial requests given the responses are best effort. So in such scenarios, we don't want to flood the console with logs.

Reply:
Can you rip out the comment and put in a comment saying what you just commented? Or change console.error to console.warn?